psamm_import/importer.py

```diff
@@ -324,13 +371,24 @@ def write_yaml_model(model, dest='.', convert_medium=True):
             yaml.safe_dump(reaction_limits, f, **yaml_args)
 
     model_d = OrderedDict([('name', model.name)])
+
+    model_files = []
+    model_files.append(('compounds', [{'include': 'compounds.yaml'}]))
     if model.biomass_reaction is not None:
         model_d['biomass'] = model.biomass_reaction
     if default_flux_limit is not None:
         model_d['default_flux_limit'] = default_flux_limit
-    model_d.update([
-        ('compounds', [{'include': 'compounds.yaml'}]),
-        ('reactions', [{'include': 'reactions.yaml'}])])
+
+    if len(subsys_files) > 0:
+        for File in subsys_files:
+            fil = (File, [{'include': File}])
+            model_files.append(fil)
+        model_files.append(
+            ('otherReactions', [{'include': 'otherReactions.yaml'}]))
```

Comment:
Maybe call the file `other_reactions.yaml`. Perhaps all the reaction files should go into a separate directory and then this file can be written to `reactions/other.yaml`.